js/admin_settings.js

```diff
@@ -19,12 +19,12 @@
  */
 
 $(document).ready(function() {
```

Comment:
DOn't use ready anymore

Reply:
okay, but -v? To get rid of jQuery totally, the post method also oughtta go.

Reply:
https://github.com/nextcloud/server/issues/20953#issue-617327468

Reply:
@skjnldsv i added a commit that gets rid of jQuery. Good?

Reply:
ah, but that was another file

Reply:
@skjnldsv When I use ```js
if (document.readyState !== 'loading') {
		fnc()
	} else {
		document.addEventListener('DOMContentLoaded', fnc)
	}```
i run into a race condition that I am always loosing, OC.Settings will never be ready by that. Of course I can fumble around with setting timeouts again…

Reply:
`document.addEventListener('DOMContentLoaded', fnc)` is enough.
Are you sure you don't want to move to a compiled package?

Reply:
Figured it out shortly afterwards :)

Not for these few lines, really.